integrations/sonarqube/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.poetry]
+name = "sonarqube"
+version = "0.1.0"
+description = "SonarQube projects and code quality analysis integration"
+authors = ["Port Team <support@getport.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+port_ocean = { version = "^0.1.0", extras = ["cli"] }
```

Comment:
you are  using an old version of ocean to generate the scaffold
id recommend using the latest version to scaffold
pull the repo from main

we should release 0.2.0 today